libraries/AP_Baro/AP_Baro_Wind.cpp

```diff
@@ -45,6 +45,22 @@ const AP_Param::GroupInfo AP_Baro::WindCoeff::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("LFT", 5, WindCoeff, yn, 0.0),
 
+    // @Param: UP
+    // @DisplayName: Pressure error coefficient in positive Z direction (up)
+    // @Description: This is the ratio of static pressure error to dynamic pressure generated by a positive wind relative velocity along the Z body axis. If the baro height estimate rises during sideways flight up (or forward with a high lean angle), then this should be a negative number. Multirotors can use this feature only if using EKF3 and if the EK3_DRAG_BCOEF_X and EK3_DRAG_BCOEF_Y parameters have been tuned.
```

Comment:
Suggest "If the baro height estimate rises above truth height during climbing flight (or forward flight with a high lean angle), then this should be a negative number."

Reply:
Thanks, fixed!